### PR TITLE
Assault

### PR DESCRIPTION
**Pull request by Glloyd**
PRing this so I make sure I don't forget any files.

### CODE REVIEW

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -107,7 +107,7 @@
 
 		if(href_list["observe"])
 
-			if(alert(src,"Are you sure you wish to observe? You will have to wait 30 minutes before being able to respawn!","Player Setup","Yes","No") == "Yes")
+			if(alert(src,"Are you sure you wish to observe? You will have to wait 20 minutes before being able to respawn!","Player Setup","Yes","No") == "Yes")
```
Comment (scrdest) [inline code/modules/mob/new_player/new_player.dm]: you var'd it, so might as well replace it with respawntime/60 to save on editing those later

File: code/modules/urist/gamemodes/infestation/aliens/lactera.dm [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+	name = "Lactera"
+	name_plural = "Lactera"
+
+	icobase = 'icons/uristmob/r_lactera.dmi'
+
+	unarmed_types = list(/datum/unarmed_attack/claws)
+	hud_type = null
+
+	has_fine_manipulation = 1
+	gluttonous = 2
```
Comment (scrdest) [inline code/modules/urist/gamemodes/infestation/aliens/lactera.dm]: are they supposed to be able to eat humans?

**Comment by scrdest** [comment]
You seem to have, in fact, missed four files for the commit.

**Comment by Glloyd** [comment]
Yeah, forgot to push some stuf before I went to work

**Comment by Glloyd** [comment]
Done for now.

**Comment by Glloyd** [comment]
Yay!

File: code/modules/urist/gamemodes/assault/equipment.dm [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+	if(health <= 0)
+		kaboom()
+
+	..()
+
+/obj/structure/assaultshieldgen/proc/kaboom()
+	for(var/obj/structure/assaultshieldgen/S in world)
+		remaininggens -= 1
+
+	if(remaininggens == 0)
```
Comment (scrdest) [inline code/modules/urist/gamemodes/assault/equipment.dm]: This being an object var is why it ends when one is broken, I suspect.

**Pull request merged**